src/pages/ExamsPage/components/AttemptList.jsx

```diff
@@ -27,6 +27,57 @@ const ExamAttemptStatusUILabels = {
   error: 'Error',
 };
 
+const StatusFilterChoices = [
+  {
+    name: 'Created',
+    value: 'created',
```

Comment:
we can use the constants we defined in the last PR for this
```suggestion
    name: ExamAttemptStatusUILabels.created,
    value: ExamAttemptStatus.created,
```